<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@838.va_3a_087b_4055b">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@2802.v5ea_628154b_c2">
    <parentIds>
      <string>17</string>
    </parentIds>
    <id>18</id>
    <startId>13</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@1192.v2d0deb_19d212">
      <startTime>1667185029773</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@1192.v2d0deb_19d212">
      <error class="java.lang.IllegalArgumentException">
        <detailMessage>/home/jenkins/code/automate_new.sh does not exist.</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.sshsteps.steps.ScriptStep$Execution.run(ScriptStep.java:74)</trace>
          <trace>org.jenkinsci.plugins.sshsteps.util.SSHStepExecution.lambda$start$0(SSHStepExecution.java:84)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions class="empty-list"/>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>